src/test/kotlin/com/ing/serialization/bfl/serde/serializers/custom/BigDecimalSerializerTest.kt

```diff
@@ -36,6 +40,20 @@ class BigDecimalSerializerTest {
         roundTrip(data, data::class)
     }
 
+    @Test
+    fun `serialize and deserialize BigDecimal ten`() {
+        val data = Data(BigDecimal.TEN)
+        roundTripInlined(data)
+        roundTrip(data, data::class)
+    }
+
+    @Test
+    fun `serialize and deserialize BigDecimal one tenth`() {
+        val data = Data(BigDecimal("0.01"))
+        roundTripInlined(data)
+        roundTrip(data, data::class)
+    }
+
```

Comment:
Again personal preference: I would merge these 2 tests (since they are testing the same functionality on different bigDecimals) and have something like:
```suggestion
    @Test
    fun `serialize and deserialize BigDecimals() {
        listOf(
             Data(BigDecimal.TEN),
             Data(BigDecimal("0.01")),
        ).forEach {
            roundTripInlined(it)
            roundTrip(it, it::class)
        }
    }

```

Reply:
Done, I even combined it with the test before these.